runtime/crab/src/pallets/bridge_dispatch.rs

```diff
@@ -184,7 +184,7 @@ impl Config<WithCrabParachainDispatch> for Runtime {
 	type AccountIdConverter = bp_crab_parachain::AccountIdConverter;
 	type BridgeMessageId = (LaneId, MessageNonce);
 	type Call = Call;
-	type CallValidator = CallValidator;
+	type CallValidator = Everything;
```

Comment:
Why?

Reply:
For para chain, it doesn't have support EVM features now, the `CallValidator` here is mainly used to validate evm calls. Thus, no validation needs in this case. See https://github.com/darwinia-network/darwinia-common/blob/main/node/runtime/pangolin/src/pallets/bridge_dispatch.rs#L175

Reply:
@xiaoch05 Do you agree with this? Since I saw https://github.com/darwinia-network/darwinia/pull/918/files.

Reply:
If we donot allow other dispatch call to be called remote, i think we need reserve this CallFilter.

Reply:
@HackFisher  Please take a review.

Reply:
More strict is better, this validator is not only for evm.

Reply:
Done, switched to stricter validation